packages/playground/compile-wordpress/build.js

```diff
@@ -3,6 +3,11 @@ import { spawn } from 'child_process';
 import yargs from 'yargs';
 
 const presets = {
+  develop: {
```

Comment:
This will keep coming up in other PRs so I should just add a pre-commit hook. I really dislike lint-staged because of how many times it put me in a detached HEAD state, but Husky alone should suffice since nx is smart enough to ignore unchanged files.